### DISCUSSION TITLE
Add custom GPT-2 Models

### DISCUSSION THREAD

**Discussion by RomeroRZ** [adieyal/sd-dynamic-prompts#381] [Q&A] [answered]
Hi, 

thanks for this awesome extension, using it all day.

i'm wondering, i have been searching where the downloaded GPT-2 models were located such a Gustavosta, Daspartho etc but can't find them.

Is there a way if i'm about to train a GPT-2 model to add them to the list ?

Thanks in advance

**Comment by adieyal** [comment, marked as answer]
Mine are downloaded here: $HOME/.cache/huggingface/hub. 

To add a new model, upload it Hugging Face and edit config/magicprompt_models.txt - if you would like your model included in dynamic prompts, send me a link when you're ready.

**Reply by RomeroRZ** [reply]
Thanks for your quick support !

I'll let you know, have a good day.
